tools/pd-control.md

```diff
@@ -245,6 +247,12 @@ The configuration above is global. You can also tune the configuration by config
     config set disable-raft-learner true        // Disable Raft learner
     ```
 
+- `cluster-version` is the version of the cluster, which is used to enable or disable some Features and to deal with the compatibility issues. By default, it is the minimum version of all normally running TiKV nodes in the cluster. Only when you need to roll back to an earlier version, you can set it manually.
```

Comment:
The last sentence:

Generally, "Only when .... can (sb)..." is an inverted sentence. It should be "Only when you need to roll back to an earlier version, can you set it manually."

But in a user document, we seldom use this sentence pattern. Suggested version:

**You can set it manually only when you need to roll back it to an earlier version.**